go.mod

```diff
@@ -57,6 +57,7 @@ require (
 	github.com/mattn/go-isatty v0.0.20 // indirect
 	github.com/mohae/deepcopy v0.0.0-20170929034955-c48cc78d4826 // indirect
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822 // indirect
+	github.com/oapi-codegen/oapi-codegen/v2 v2.4.1 // indirect
```

Comment:
hm, why was this necessary?

Reply:
I just ran the prepare source 🤔🤷
I'll check again in detail

Reply:
`go generate ./...` creates this - not sure if this should be committed?
I remove the commit for now